assets/aml-benchmark/components/src/prompt_crafter/package/prompt_factory.py

```diff
@@ -190,6 +191,9 @@ def process_row(self, row: Dict, index: int) -> Dict:
         if self.output_pattern is not None:
             output_data['completion'] = self.get_label_from_output_pattern(row)
 
+        if self.ground_truth_column_name is not None and self.ground_truth_column_name in row:
+            output_data['ground_truth_column_name'] = row[self.ground_truth_column_name]
```

Comment:
Instead of adding a column with name `ground_truth_column_name`, can we use `ground_truth`?

Reply:
I think we should use the value that was passed in the `ground_truth_column_name` as the column name for ground_truth in the output file.

Reply:
@iamrk04 , just wondering if there might be odds of a collision in name in certain scenarios. We generate prompt and completion columns. If one of these is used in `ground_truth_column_name`, it would be an invalid scenario for the component.

Reply:
makes sense, let's hardcode this to `ground_truth`. Thanks!